resolwe/flow/serializers/fields.py

```diff
@@ -41,7 +41,10 @@ def model_name(self):
 
     def to_internal_value(self, data):
         """Convert to internal value."""
-        if data.get('id', None) is not None:
+        # Allow None values only in case field is not required.
+        if 'id' in data and data['id'] is None and not self.required:
```

Comment:
Is this `required` check needed or is it handled at validation step?

Reply:
Not required. Removed. What bothers me is that if a required field is updated to None (for example Data.process=None), the response is http status code 204, but the field stays unchanged. Should we raise error on such requests?